src/installToolsInRemote.ts

```diff
@@ -7,7 +7,7 @@
 import * as os from 'os';
 import * as path from 'path';
 import * as shell from 'shelljs';
-import { downloadFileAndCreateSha256 } from '../build/bundle-tools';
+import { downloadFileAndCreateSha256 } from './downloadBinaries/downloadBinaries';
```

Comment:
can you update some different name to folder/class? currently both names were same